source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -563,14 +538,236 @@ the Kubernetes cluster.
+
+The command requires values for each of the following:
+
+- :mc:`~kubectl minio tenant create TENANT_NAME`
+    
+  The name to use for the new tenant.
+
+- :mc:`~kubectl minio tenant create --capacity`
+
+  The total raw storage size for the Tenant across all volumes. 
+  Specify both the total storage size *and* the :guilabel:`Unit` of that storage. 
+  All storage units are in SI values, e.g. :math:`Gi = GiB = 1024^3` bytes.
+     
+  For example, 16 Ti for 16 Tebibytes.
+
+- :mc:`~kubectl minio tenant create --servers`
+
+  The total number of MinIO server pods to deploy in the Tenant.
+    
+  The Operator by default uses pod anti-affinity, such that the Kubernetes cluster *must* have at least one worker node per MinIO server pod.
+
+- :mc:`~kubectl minio tenant create --volumes`
+
+  The total number of storage volumes (Persistent Volume Claims). 
+  The Operator generates an equal number of PVC *plus two* for supporting Tenant services (Metrics and Log Search).
+       
+  The total number of persistent volume claims (``PVC``) per server is determined by dividing the number of volumes by the number of servers.
+  The storage available for each ``PVC`` is determined by dividing the capacity by the number of volumes. 
+
+  The generated claims have pod selectors so that claims are only made for volumes attached to node running the pod.
+
+  If the number of volumes exceeds the numnber of persistent volumes available on the cluster, ``MinIO`` hangs until the number of persistent volumes are available.
+  
+- :mc:`~kubectl minio tenant create --namespace`
+
+  Each MinIO tenant requires its own ``namespace``.
+
+  When not specified, the MinIO Operator defaults to use ``minio``.
+  To specify a different namespace, use the :mc:`~kubectl minio tenant create --namespace` flag.
+
+  The namespace must already exist in the Kubernetes cluster.
+  Run ``kubectl create ns <new_namespace>`` to add one.
+
+.. _create-tenant-cli-determine-additional-options:
+
+2) Determine Additional Optional Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-STUB
\ No newline at end of file
+You can further customize your tenant while executing the command by including any or all of the *optional* flags.
+Determine whether to use each of the following and the value to set:
+
+- :mc:`~kubectl minio tenant create --image`
+           
+  Customize the ``minio`` image to use.
+  
+  By default, the Operator uses the release image available at the time of the Operator's release.
+  To specify a different MinIO version for the tenant, such as the latest available, use the :mc:`~kubectl minio tenant create --image` flag.
+
+  See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.
+
+- :mc:`~kubectl minio tenant create --image-pull-secret`
+           
+  If required, specify the secret to use when pulling the ``minio`` image.
```

Comment:
We should specify if at all possible why this would be required. I _think_ in this case it's if the specified image repository requires a password for access.

Reply:
For custom container registries. Clarified.